src/parser/creative_linear_parser.js

```diff
@@ -42,24 +42,33 @@ export function parseCreativeLinear(creativeElement, creativeAttributes) {
     'VideoClicks'
   );
   if (videoClicksElement) {
-    creative.videoClickThroughURLTemplate = parserUtils.parseNodeText(
-      parserUtils.childByName(videoClicksElement, 'ClickThrough')
+    const videoClickThroughElement = parserUtils.childByName(
+      videoClicksElement,
+      'ClickThrough'
     );
+    if (videoClickThroughElement) {
+      creative.videoClickThroughURLTemplate = {
```

Comment:
should we assign `creative.videoClickThroughURLTemplate = null` if there is no `videoClickThroughElement`? Maybe it would be clearer to the user that it is not a parsing problem, just that there was no value to parse.

Reply:
also please be aware these changes break tracking in `vast_tracker.js` : https://github.com/dailymotion/vast-client-js/blob/master/src/vast_tracker.js#L374
The file will have to be updated as well to ensure all the tracking works

Reply:
Done